src/components/tabs/_tabs.scss

```diff
@@ -84,25 +84,60 @@
         color: govuk-colour("black");
         background-color: govuk-colour("light-grey", $legacy: "grey-4");
         text-align: center;
-        text-decoration: none;
+        text-decoration: underline;
+
+        @include govuk-if-ie8 {
+          // IE8 doesn't support `box-shadow` so add an actual border that is
+          // modified to indicate state.
+          border-top: 4px solid transparent;
+        }
+
+        &:hover {
+          box-shadow: inset 0 4px 0 0 govuk-colour("light-blue");
+
+          @include govuk-if-ie8 {
+            border-top-color: govuk-colour("light-blue");
+          }
+        }
 
         &--selected {
+          $top-border-width: 1px;
+          $top-border-adjustment: 1px;
+
+          @include govuk-if-ie8 {
+            // IE8 already has a top border so no adjustment is needed
+            $top-border-width: 4px;
+            $top-border-adjustment: 0;
+          }
+
           margin-top: - govuk-spacing(1);
           margin-bottom: -1px;
 
           // 1px is compensation for border (otherwise we get a 1px shift)
-          padding-top: govuk-spacing(3) - 1px;
+          padding-top: govuk-spacing(3) - $top-border-adjustment;
           padding-right: govuk-spacing(4) - 1px;
           padding-bottom: govuk-spacing(3) + 1px;
           padding-left: govuk-spacing(4) - 1px;
 
-          border: 1px solid $govuk-border-colour;
-          border-bottom: 0;
-          color: govuk-colour("black");
+          border-top: $top-border-width solid transparent;
+          border-right: 1px solid $govuk-border-colour;
+          border-left: 1px solid $govuk-border-colour;
+
+          color: $govuk-text-colour;
           background-color: govuk-colour("white");
+          box-shadow: inset 0 4px 0 0 $govuk-link-colour;
+          text-decoration: none;
+
+          @include govuk-if-ie8 {
+            border-top-color: $govuk-link-colour;
+          }
 
           &:focus {
-            background-color: transparent;
+            box-shadow: inset 0 4px 0 0 $govuk-focus-colour, inset 0 8px 0 0 govuk-colour("black");
+
+            @include govuk-if-ie8 {
```

Comment:
In the accordion component I've put a border + a smaller box-shadow for all browsers, which allows you to avoid the special IE8 wrapper stuff.

Not sure if it'd work here, but means it fallsback automatically without conditional stylesheets.

Reply:
Thanks @nickcolley. Happy to revisit this after the audit and there's also this to consider: https://github.com/alphagov/govuk-frontend/issues/1327